css-positioning/absolute.css

```diff
@@ -0,0 +1,41 @@
+.meme {
+  width: 300px;
+  height: 300px;
+  background-image: url("images/one-does-not-simply.jpg");
+  background-size: cover;
+
+  /* your code here */
```

Comment:
you should set `position` here to something so that all the descendant elements will be positioned against this "non-static" element.